<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavManagerDialog</class>
 <widget class="QWidget" name="NavManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	font-family:&quot;微软雅黑&quot;;
}

QWidget#widget_4
{
	border-left:1px solid #919191; 
	border-bottom:1px solid #919191 ;
    border-right:1px solid #919191;
}

QPushButton#pbt_addNav
{
	border-top:1px ;
    border-bottom:1px ;
	border-right:1px solid #919191;
	image: url(:/Images/icon/nav_add.png);    
}

QPushButton#pbt_addNav:hover
{
	image: url(:/Images/icon/nav_add_hover.png); 
}

QPushButton#pbt_deleteNav
{
	border-top:1px ;
    border-bottom:1px ;
	border-right:1px solid #919191;
	image: url(:/Images/icon/nav_sub.png); 
}
QPushButton#pbt_deleteNav:hover
{
	image: url(:/Images/icon/nav_sub_hover.png); 
}

QPushButton#pbt_applyNav,QPushButton#pbt_selectnone
{
	background:#1ba9ba;
	color:#efefef;
	border:none;
}
QPushButton#pbt_applyNav:hover,QPushButton#pbt_selectnone:hover
{
	background:#3dbbca;
}

QPushButton#pbt_adver
{
	color: #666666;
	background-color: #FFFFFF;
	border:none;
}
QPushButton#pbt_adver:hover
{
	color: #FFFFFF;
	background-color: #3dbbca;
}
QRadioButton#radbtn_pb,QRadioButton#radbtn_xmpp
{
	 font-size: 12px;
	font-weight:bold;
}
QLabel#label_yu
{
	font-size: 12px;
	font-weight:bold;
}
QWidget#widget_2
{
	background-color: #E2E2E2;
}
QLineEdit{
	font-size: 12px;
	font-weight:bold;
    background-color: #00000000;
    border:none;
}
QLineEdit:hover{
    border:1px solid #999999;
}
QLineEdit:focus{
    background-color: #FFFFFF;
    border-width: 1px;
    border-style: solid;
    border-color: #1BA9BA;
}

QLineEdit:disabled{
    background-color: #ECECEC;
    border:none;
}

QTextEdit{
	font-size: 12px;
	font-weight:bold;
    background: #FFFFFF;
    border:none;
}

QTextEdit:focus{
    background-color: #FFFFFF;
    border-width: 1px;
    border-style: solid;
    border-color: #1BA9BA;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="shadowbg" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QWidget" name="shadowinnerbg" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="TitlebarWidget" name="titlebar" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_8" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <widget class="QRadioButton" name="radbtn_xmpp">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>10</y>
              <width>89</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>xmpp协议</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="radbtn_pb">
            <property name="geometry">
             <rect>
              <x>290</x>
              <y>10</y>
              <width>89</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>pb协议</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <widget class="QWidget" name="widget_2" native="true">
            <property name="geometry">
             <rect>
              <x>210</x>
              <y>10</y>
              <width>300</width>
              <height>313</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_6" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>73</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label_erCode">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>120</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>120</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>72</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_5" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>31</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>30</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="minimumSize">
                   <size>
                    <width>31</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>名称:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_title">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
               <zorder>lineEdit_title</zorder>
               <zorder>label_2</zorder>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_9" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>31</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>30</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="minimumSize">
                   <size>
                    <width>31</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>域名:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_yu">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_7" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>91</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>91</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>3</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="minimumSize">
                   <size>
                    <width>54</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>54</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>导航地址:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextEdit" name="textEdit_nav">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>80</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="widget_3" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>10</y>
              <width>201</width>
              <height>371</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="QListWidget" name="listWidget"/>
             </item>
             <item>
              <widget class="QWidget" name="widget_4" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>2</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="pbt_addNav">
                  <property name="minimumSize">
                   <size>
                    <width>23</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>23</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>新添加</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pbt_deleteNav">
                  <property name="minimumSize">
                   <size>
                    <width>23</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>23</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>删除项</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>150</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QPushButton" name="pbt_applyNav">
            <property name="geometry">
             <rect>
              <x>407</x>
              <y>338</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>应用</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="pbt_adver">
            <property name="geometry">
             <rect>
              <x>230</x>
              <y>338</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>高级</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="pbt_selectnone">
            <property name="geometry">
             <rect>
              <x>320</x>
              <y>340</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>清除所有</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitlebarWidget</class>
   <extends>QWidget</extends>
   <header>TitlebarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
